File: Projects/server/Contracts/Requests/SignUpRequest.cs
namespace server.Contracts.Requests
{
    public class SignUpRequest
    {
        public string Email { get; set; } = null!;
        public string Phone { get; set; } = null!;
        public string LastName { get; set; } = null!;
        public string FirstName { get; set; } = null!;
        public string? Patronymic { get; set; }
        public DateOnly Birth { get; set; }
        public string Password { get; set; } = null!;
        public string? DriveLicense { get; set; }
        public byte RoleId { get; set; }
    }
}

File: Projects/DataProviderApp/app/src/main/java/com/example/dataproviderapp/dto/requests/SignUpRequest.kt
package com.example.dataproviderapp.dto.requests

import java.time.LocalDate

data class SignUpRequest(
    val email: String,
    val phone: String,
    val lastName: String,
    val firstName: String,
    val patronymic: String?,
    val birth: LocalDate,
    val password: String,
    val driveLicense: String?,
    val roleId: UByte
)
